### PR TITLE
base: UEFI Secure Boot Provisioning

### PR DESCRIPTION
**Pull request by ldts**
Simplify the Secure Boot key provisioning process by adding a systemd-boot entry wich uses the [efitools](https://github.com/mjg59/efitools.git) EFI program [LockDown.efi](https://github.com/mjg59/efitools/blob/master/LockDown.c).

image

On selecting that entry, the keys are provisioned after which the system  resets.

Users can then boot into Linux with a secure boot enabled chain.

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]

**Comment by ldts** [comment]
Using the default keys provided with lmp-tools on tag 94 will fail with output:

image

**Comment by ldts** [comment]
This PR can be merged at any time - no dependencies. It just needs valid keys.

File: meta-lmp-base/recipes-bsp/efitools/efitools/0001-lockdown-systemd-boot-conf.patch
Comment (ricardosalveti) [inline meta-lmp-base/recipes-bsp/efitools/efitools/0001-lockdown-systemd-boot-conf.patch:22]: You don't need a patch for this, you can add the file directly at SRC_URI.

Comment (ldts) [reply meta-lmp-base/recipes-bsp/efitools/efitools/0001-lockdown-systemd-boot-conf.patch:22]: sure, I thought the patch will be better, anyway, let me update

**Comment by ldts** [comment]
@rsalveti anything else you want to add before I merge

**Comment by ricardosalveti** [comment]
Nops, but don't merge just yet, we're in the process of releasing v94.1, holding on merges until we're good with it.

**Comment by ldts** [comment]
> Nops, but don't merge just yet, we're in the process of releasing v94.1, holding on merges until we're good with it.

just ping me whenever

**Pull request merged**
